### DISCUSSION TITLE
keyball61: トラックボール無し側(左で使用) ジャンパ部分のパッド剥離

### DISCUSSION THREAD

**Discussion by k-yoshiyuki** [Yowkees/keyball#218] [Q&A] [answered]
keyball61のトラックボール無し側(左で使用)のジャンパ部分の緑四角で囲んだ部分のパッドが剥離してしまいました。
回路図確認しましたが、あっているか不安ですので確認させてください。
赤丸で囲んだ2点を直接導線で接続すれば、修復可能でしょうか？

お手数おかけしますが、ご確認のほどよろしくお願いします。
![image](https://user-images.githubusercontent.com/93636102/232250541-45a88792-e5b8-40d0-883e-406f9e890edf.png)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
はい。赤丸２点で合っています。
ご対応よろしくお願いいたします。

**Reply by k-yoshiyuki** [reply]
ご回答ありがとうございます。
承知いたしました。赤丸2点でジャンパー線の接続してみます。
